src/ProjectDeploy/index.tsx

```diff
@@ -38,11 +38,15 @@ export function ProjectDeploy() {
         behavior: "smooth",
       });
     },
-    onComplete: () => {
+    onComplete: (data: string) => {
+      const { buildOriginalDomain } = JSON.parse(
+        data.match(/\{([^}]+)\}/g)?.pop() as string,
+      );
```

Comment:
```suggestion
      const regExp = /\{([^}]+)\}/g;
      const resultMessage = data.match(regExp)?.pop() as string;
      const originalBuildDomain = JSON.parse(resultMessage);
```

onComplete에서는 문자열 중에 regExp로 특정 데이터를 추출하고, JSON.parse로 데이터를 변환하는데, 이 동작이 한 번에 이루어져서 가독성 유지보수성이 좋지 않을 수도 있을 것 같아요~ 이렇게 수정되면 어떨까요?

Reply:
regExp로 분리해도 설명이 되지 않을 것 같아 한 번에 했는데 분리하는 게 더 좋을 것 같네요 적용하겠습니다~